pages/[address].tsx

```diff
@@ -0,0 +1,385 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { GetServerSideProps, InferGetServerSidePropsType } from "next";
+import Image from "next/image";
+import { ethers } from "ethers";
+
+import {
+  Layout,
+  Hero,
+  Card as UnstyledCard,
+  Value,
+  Link as UnstyledLink,
+  About,
+  Information as UnstyledInformation,
+  SynthPlaceholderIcon,
+  GettingStarted,
+  LiveIndicator,
+  MaxWidthWrapper,
+  Table,
+} from "../components";
+import {
+  formatMillions,
+  QUERIES,
+  getContenfulClient,
+  ContentfulSynth,
+  formatContentfulUrl,
+  errorFilter,
+} from "../utils";
+import LeftArrow from "../public/icons/arrow-left.svg";
+import UnstyledRightArrow from "../public/icons/arrow-right.svg";
+import UnstyledExternalLink from "../public/icons/external-link.svg";
+import { client, EmpState, EmpStats } from "../utils/umaApi";
+
+const BackAction = () => {
+  return (
+    <ActionWrapper href="/">
+      <LeftArrow />
+      Back to projects
+    </ActionWrapper>
+  );
+};
+
+const ActionWrapper = styled(UnstyledLink)`
+  color: var(--white);
+  font-size: ${14 / 16}rem;
+  display: flex;
+  align-items: center;
+  gap: 10px;
+
+  margin-bottom: 16px;
+`;
+
+type Emp = ContentfulSynth & EmpStats & EmpState;
+export const getServerSideProps: GetServerSideProps = async (ctx) => {
```

Comment:
theres a lot going on in here, not sure i totally understand the function outisde of joining a bunch of sources